README.md

```diff
@@ -189,7 +189,7 @@ uid           [ unknown] root <root@localhost>
 The fingerprint here is `9F89C5F69A10281A835014B09C3DC61F752087EF`.
 </details>
 
-Your `age` public key or GPG fingerprint can be written to your [`.sops.yaml`](https://github.com/getsops/sops#using-sops-yaml-conf-to-select-kms-pgp-and-age-for-new-files) in the root of your configuration directory or repository:
+Your `age` public key or GPG fingerprint can be written to your [`.sops.yaml`](https://github.com/getsops/sops#using-sopsyaml-conf-to-select-kms-pgp-and-age-for-new-files) in the root of your configuration directory or repository:
```

Comment:
The old link https://github.com/getsops/sops#using-sops-yaml-conf-to-select-kms-pgp-and-age-for-new-files works for me but not the new one: https://github.com/getsops/sops#using-sopsyaml-conf-to-select-kms-pgp-and-age-for-new-files

Reply:
Yes I can confirm the old link seems to be working again. I did some digging around and it seems that GitHub changed how they render `.rst` documents which caused some things (including the Table of Contents) to not render correctly. 
I guess this changed some of the heading links too.

For reference:
- [Archive.org of the improperly rendered .rst](https://web.archive.org/web/20240518223021/https://github.com/getsops/sops?tab=readme-ov-file#using-sopsyaml-conf-to-select-kms-pgp-and-age-for-new-files)
- [People reporting issues about improper rendering](https://github.com/orgs/community/discussions/86715)
- https://github.com/jgm/pandoc/issues/9805 - This may have fixed it